src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/OcrValidator.java

```diff
@@ -103,10 +103,11 @@ private void handleValidationResponse(
         log.info("Validation result for envelope {}: {}", envelope.zipFileName, res.status);
         switch (res.status) {
             case ERRORS:
+                String message = "OCR validation service returned OCR-specific errors. "
+                    + "Document control number: " + docWithOcr.documentControlNumber + ". "
+                    + "Envelope: " + envelope.zipFileName + ".";
                 throw new OcrValidationException(
-                    "OCR validation service returned OCR-specific errors. "
-                        + "Document control number: " + docWithOcr.documentControlNumber + ". "
-                        + "Envelope: " + envelope.zipFileName + "."
+                    message, message + " Errors: " + res.errors
```

Comment:
Would be better just to have errors in detailed message. Also not sure if writing list (res.errors) will be readable enough.

Reply:
we need document control and filename, it will print the error  like this: ["", ""]